### PR TITLE
👷 Add use methodInt and Extract "USE" to a global const variable

### PR DESCRIPTION
**Pull request by kiyonlin**
Ref #589

### CODE REVIEW

File: utils.go [hunk 1]
```diff
@@ -270,6 +272,7 @@ const (
 	MethodConnect = "CONNECT" // RFC 7231, 4.3.6
 	MethodOptions = "OPTIONS" // RFC 7231, 4.3.7
 	MethodTrace   = "TRACE"   // RFC 7231, 4.3.8
+	methodUse     = "USE"
```
Comment (renanbastos93) [inline utils.go:275]: Maybe we should change it.
```suggestion
	MethodUse     = "USE"
```

Comment (kiyonlin) [reply utils.go:275]: Now it’s just used inside the package.

Comment (renanbastos93) [reply utils.go:275]: it's okay, hehehe my mistake

**Comment by Fenny** [comment]
Awesome work, thank you ❤️

**Pull request merged**